client/me/help/main.jsx

```diff
@@ -207,15 +203,20 @@ class Help extends React.PureComponent {
 		}
 
 		return (
-			<HelpTeaserButton
+			<CompactCard
+				className="help__support-link"
+				href={ localizeUrl( 'https://wordpress.com/webinars' ) }
 				onClick={ this.trackCoursesButtonClick }
-				href="https://wordpress.com/webinars"
-				target="_blank"
-				title={ this.props.translate( 'Courses' ) }
-				description={ this.props.translate(
-					'Learn how to make the most of your site with these courses and webinars'
-				) }
-			/>
+				target="__blank"
+			>
+				<Gridicon icon="chat" size={ 36 } />
+				<div className="help__support-link-section">
+					<h2 className="help__support-link-title">{ this.props.translate( 'Webinars' ) }</h2>
+					<p className="help__support-link-content">
+						{ this.props.translate( 'Make the most of your site with courses and webinars.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Learn how to make the most of your site with these courses and webinars' )` *ES Score: 7*